packages/components/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -103,6 +97,11 @@ export const SegmentedControl: FC<SegmentedControlProps> = ({
       textAlign: 'center',
     }
 
+    const a11yListPosition = t('listPosition', {
```

Comment:
Nitpicky, but old position of setting the const alongside the a11yLabel made more sense readability-wise